packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -0,0 +1,189 @@
+            WARNING - The wafermap is still in development and considered
+            experimental. It is not recommended for application use.
+        </div>
+        <nimble-wafer-map
+            colorsScaleMode="${x => x.colorsScaleMode}"
+            dieLabelsHidden="${x => x.dieLabelsHidden}"
+            dieLabelsSuffix="${x => x.dieLabelsSuffix}"
+            maxCharacters="${x => x.maxCharacters}"
+            orientation="${x => x.orientation}"
+            quadrant="${x => x.quadrant}"
+            :colorScale="${x => x.colorScale}"
+            :dies=${x => x.dies}
+            :highlightedValues="${x => x.highlightedValues}"
+        >
+        </nimble-wafer-map>
+        <style class="code-hide">
+            #usage-warning {
+                color: red;
+                font: var(${bodyFont.cssCustomProperty});
+            }
+        </style>
+    `),
+    args: {
+        dies: wafermapDieSets[0],
+        quadrant: WaferMapQuadrant.bottomLeft,
+        orientation: WaferMapOrientation.left,
+        colorScale: waferMapColorsScaleSets[0],
+        maxCharacters: 4,
+        dieLabelsHidden: false,
+        dieLabelsSuffix: '',
+        colorsScaleMode: WaferMapColorsScaleMode.linear,
+        highlightedValues: highLightedValueSets[0]
+    },
+    argTypes: {
+        dies: {
+            description:
+                'Represents the input data, an array of `WaferMapDie`, which will be renedered by the wafer map',
+            options: ['set1', 'set2'],
+            control: {
+                type: 'select',
+                labels: {
+                    set1: 'Dies Set 1',
+                    set2: 'Dies Set 2'
+                }
+            },
+            defaultValue: 'set1',
+            mapping: {
+                set1: wafermapDieSets[0],
+                set2: wafermapDieSets[1]
+            }
+        },
+        quadrant: {
+            description:
+                'Represents the orientation of the dies on the wafer map',
+            options: Object.values(WaferMapQuadrant),
+            control: {
+                type: 'select',
+                labels: {
+                    [WaferMapQuadrant.bottomLeft]: 'BottomLeft',
+                    [WaferMapQuadrant.bottomRight]: 'BottomRight',
+                    [WaferMapQuadrant.topLeft]: 'TopLeft',
+                    [WaferMapQuadrant.topRight]: 'TopRight'
+                }
+            }
+        },
+        orientation: {
+            description: 'Notch orientation',
+            options: Object.values(WaferMapOrientation),
+            control: {
+                type: 'select',
+                labels: {
+                    [WaferMapOrientation.left]: 'Left',
+                    [WaferMapOrientation.top]: 'Top',
+                    [WaferMapOrientation.right]: 'Right',
+                    [WaferMapOrientation.bottom]: 'Bottom'
+                }
+            }
+        },
+        maxCharacters: {
+            description:
```

Comment:
applied

Reply:
@DStavilaNI I think there was a misunderstanding. The goal is so the names in the story align with the names in the user html example:
![image](https://user-images.githubusercontent.com/1588923/204602902-c589d6c1-71ec-4953-8f8f-1620c2c5f558.png)

Currently the html that is rendered is also incorrect:

![image](https://user-images.githubusercontent.com/1588923/204603400-a80e687c-1ad5-46b2-886f-4466d0fd18c7.png)

I'd expect the attribute names in storybook to look like `max-characters` as my comment described and not `maxcharacters` as your change shows. For such a big deviation from comments it may be worth asking for a clarification.